actions/merger/entrypoint.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+
+set -e
+
+# config
+GITHUB_REPO=${GITHUB_REPO:-jade-data-repo}
+GITHUB_USER_EMAIL=${GITHUB_USER_EMAIL:-broadbot@broadinstitute.org}
+GITHUB_USER_NAME=${GITHUB_USER_NAME:-broadbot}
+GITHUB_WORKSPACE=${GITHUB_WORKSPACE}
+COMMIT_MESSAGE="${COMMIT_MESSAGE:-"commit merged via github action"}"
+MERGE_BRANCH=${MERGE_BRANCH:-develop}
+SWITCH_DIRECTORIES=${SWITCH_DIRECTORIES:-"false"}
+
+if ${SWITCH_DIRECTORIES} ; then
+  echo "[INFO] change directory to ${GITHUB_WORKSPACE}/${GITHUB_REPO}"
+  cd "${GITHUB_WORKSPACE}"/"${GITHUB_REPO}"
+fi
+
+echo "[INFO] git config email to ${GITHUB_USER_EMAIL}"
```

Comment:
Not sure what the convention is for jade scripts, but if you want to echo each command before it's executed you can use `set -x`. That avoids any case where you might mistype the `echo` or not keep it up to date.

In this case I would add `set -x` before the first `git` command and `set +x` after the last one.